Assets/Scripts/Customers/Customer.cs

```diff
@@ -0,0 +1,24 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class Customer : MonoBehaviour
+{
+    [SerializeField] private float _walkSpeed;
```

Comment:
Can we put `walkSpeed` under a scriptable object named `CustomerConfiguration`? You can refer to the SO here and access the `walkSpeed`.